CONTRIBUTING.md

```diff
@@ -8,23 +8,34 @@
 
 ## Prerequisites
 
-Before you start contributing, review these [basic guidelines](https://www.cockroachlabs.com/docs/stable/contribute-to-cockroachdb.html) on finding a project, determining its complexity, and learning what to expect in your collaboration with the Cockroach Labs team.
+Before you start contributing, review these [basic
+guidelines](https://www.cockroachlabs.com/docs/stable/contribute-to-cockroachdb.html)
+on finding a project, determining its complexity, and learning what to
+expect in your collaboration with the Cockroach Labs team.
 
 ## Getting and Building
 
 1. Install the following prerequisites, as necessary:
-   - A C++ compiler that supports C++11. Note that GCC prior to 6.0 doesn't
-    work due to https://gcc.gnu.org/bugzilla/show_bug.cgi?id=48891
-   - A Go environment with a recent 64-bit version of the toolchain. Note that
-    the Makefile enforces the specific version required, as it is updated
-    frequently.
-   - Git 1.9+
-   - Bash (4+ is preferred)
-   - GNU Make (3.81+ is known to work)
-   - CMake 3.1+
-   - Autoconf 2.68+
-   - Optional: NodeJS 6.x and Yarn 0.22.0+. Required when compiling protocol
-    buffers.
+
+   - Either a GNU/Linux system with a working Docker install, so you
```

Comment:
Can you drop "GNU/Linux" so it reads "Either a system with a working Docker install..."? Windows and macOS can use Docker too.

Reply:
Done